.github/workflows/ci.yaml

```diff
@@ -102,42 +94,33 @@ jobs:
         # the login password doesn't matter, but the keyring must be unlocked for the tests to work
         run: gnome-keyring-daemon --components=secrets --daemonize --unlock <<< 'foobar'
 
-      - name: Run tests with rust crypto
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          # run tests single-threaded to avoid race conditions
-          args: --features=rt-tokio-crypto-rust -- --test-threads=1
+      - name: Run tests with RustCrypto
+        # run tests single-threaded to avoid race conditions
+        run: cargo test --features=rt-tokio-crypto-rust -- --test-threads=1
 
-      - name: Run tests with openssl crypto
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          # run tests single-threaded to avoid race conditions
-          args: --features=rt-tokio-crypto-openssl -- --test-threads=1
+      - name: Run tests with OpenSSL crypto
+        # run tests single-threaded to avoid race conditions
+        run: cargo test --features=rt-tokio-crypto-openssl -- --test-threads=1

```

Comment:
Do you think it’d make sense to use a matrix for the features?

Reply:
Yeah, that's a good idea. This testing was originally written before this crate had a lot of features and hasn't stayed up to date.

Reply:
I have opened https://github.com/hwchen/secret-service-rs/issues/77 to track this work. I would like to merge this without further restructuring since its blocking other PRs. Thanks again for the suggestion/callout.